.github/workflows/python-ci-docs.yml

```diff
@@ -24,7 +24,19 @@ on:
     branches:
       - 'main'
   pull_request:
-
+    paths:
+    - '**'
+    - '!.github/ISSUE_TEMPLATE/**'
+    - '!.github/workflows/*'
+    - '.github/workflows/python-ci-docs.yml'
```

Comment:
the above two ignores everything under `.github/workflows` except the current file.

Reply:
nit, this github action is super lightweight. wydt about reverting to the original and just let everything run?